### PR TITLE
recap release 1.2.0

### PR DESCRIPTION
**Pull request by tonyskapunk**
- Use of modern tools for netstat reports, `net-tools` are now deprecated. (#107)
  - https://bugzilla.redhat.com/show_bug.cgi?id=1496151
- Deprecated the use of `OPTS_CURL` in favor of `OPTS_LINKS` (#125)
- Default changed for `OPTS_STATUSURL` (#125)
- Deprecated the use of `MAXLOAD`. (#123)
- Timeout added to cronjobs with a default of 5m for most cases. (#123)
- Dependencies:
  - Removed:`bc`, `net-tools`.
  - Added: `links`, `iproute`/`iproute2`.

---

cc for review and/or comments: @carlwgeorge @man-chung  @terackspace @LukeHandle @Vascom

### CODE REVIEW

File: src/recaptool [hunk 1]
```diff
@@ -40,100 +40,100 @@ if [[ -r "${CONFFILE}" ]]; then
...
+while [[ ${#} -gt 0 ]]; do
+	case "${1}" in
     -V|--version)
       echo "${_VERSION}"
       exit 0
       ;;
 		-d)
-			if [[ -d $2 ]]; then
-				BASEDIR=$2
+			if [[ -d ${2} ]]; then
```
Comment (terackspace) [inline src/recaptool]: Should this be quoted in case the directory contains a space? (normal bash stuff)

**Comment by terackspace** [comment]
I haven't had a chance to run the code, but reviewing the commits it feels good other than that one minor item I spotted about quotes.

File: src/recaptool (hunk 1 above)
Comment (tonyskapunk) [reply src/recaptool]: Ahh good point!, I made the fix in this PR #133 133

**Comment by tonyskapunk** [comment]
Releasing 1.2.0 and tagging.

**Pull request merged**